docs/storage-abstraction.md

```diff
@@ -47,157 +69,58 @@ You can create a JSON file emoney-storage.json with the JSON below.
 }
 ```
 
-Note that the `transaction` field is set to `false`, which indicates you use this table with the Storage API.
+To apply the schema, go to the [`scalardb Releases`](https://github.com/scalar-labs/scalardb/releases) page and download the ScalarDB Schema Loader that matches the version of ScalarDB that you are using to the `getting-started` folder.
```

Comment:
```suggestion
To apply the schema, go to the [ScalarDB Releases](https://github.com/scalar-labs/scalardb/releases) page and download the ScalarDB Schema Loader that matches the version of ScalarDB that you are using to the `getting-started` folder.
```

Reply:
I think we have a similar sentence in other docs, so I'll update those to match this style later.